sroka/api/google_ad_manager/gam_api.py

```diff
@@ -92,3 +92,65 @@ def get_data_from_admanager(query, dimensions, columns, start_date, end_date, cu
     with gzip.open(report_file.name) as file:
         data = pd.read_csv(file)
     return data
+
+
+def get_users_from_admanager(query, dimensions, network_code=None):
+    
+    user_df = pd.DataFrame()
+    
+    dimensions_df = pd.DataFrame()
+        
+    #.Where() handles filtering with admanager method, while statement_query handles queries with WHERE clause
+    
+    statement_query = query.replace("WHERE ", "")
```

Comment:
Please add handling of "Where" and "where" as well (not only capitalised letters). Currently an error is thrown:
`PublisherQueryLanguageSyntaxError.UNPARSABLE`

Reply:
Done, I've added .upper() to ensure that Where is always in upper case